cluster/images/etcd/rollback/rollback.go

```diff
@@ -26,7 +26,10 @@ import (
 	"strings"
 	"time"
 
-	wal237 "k8s.io/kubernetes/third_party/forked/etcd237/wal"
+	// Uncomment when you want to rollback to 2.2.1 version.
+	oldwal "k8s.io/kubernetes/third_party/forked/etcd221/wal"
+	// Uncomment when you want to rollback to 2.3.7 version.
+	// oldwal "k8s.io/kubernetes/third_party/forked/etcd237/wal"
```

Comment:
yea. it is your call. alternatively, we can document how to change wal version clearly (and why we do not provide a flag).